### ISSUE TITLE
PyPI availability of lag-llama package

### ISSUE THREAD

**Issue by macarw** [time-series-foundation-models/lag-llama#98]
Hi lag-llama team,

Thank you for developing this project, it's a highly valuable resource. 
I wasn't able to find the lag-llama package on PyPI, which makes it a bit challenging to integrate into projects using tools like Poetry.

Could you let me know if there are any plans to publish lag-llama to PyPI, or if it's already available under a different name?

Best regards.

**Comment by ashok-arjun** [comment]
Hi, you can install it with`pip install git+https://github.com/time-series-foundation-models/lag-llama.git`.

**Comment by macarw** [comment]
Thank you for the quick response. I’m aware that the package can be installed directly from the repository. I was just curious if there are any plans to publish the project on PyPI in the future, as my team and I are encountering some challenges with security audits.

**Comment by ashok-arjun** [comment]
Yes, I can publish this in PyPI if it's really required for you. Can you let me know if they've been resolved?

**Comment by macarw** [comment]
We're still encountering the issue. We’re eager to integrate lag-llama into our project, but the lack of availability on PyPI is causing problems with pip-audit, which complicates our dependency management and affects our security controls. 
Thank you again for your support. I truly appreciate your willingness to publish the library on PyPI and I’m looking forward to its release.
